render-wasm/src/shapes/strokes.rs

```diff
@@ -46,10 +46,18 @@ pub struct Stroke {
     pub style: StrokeStyle,
     pub cap_end: StrokeCap,
     pub cap_start: StrokeCap,
-    pub kind: StrokeKind,
+    kind: StrokeKind,
 }
 
 impl Stroke {
+    pub fn kind(&self, is_open: bool) -> StrokeKind {
+        if is_open {
+            StrokeKind::CenterStroke
+        } else {
+            self.kind.clone()
```

Comment:
If we make `StrokeKind` derive `Copy` as well, we don't need to do a `.clone` here.

Reply:
Thanks! :)